tests/benchdnn/ip/ip.cpp

```diff
@@ -278,7 +278,16 @@ void skip_invalid_prb(const prb_t *prb, res_t *res) {}
 
 void setup_cmp(compare::compare_t &cmp, const prb_t *prb, data_kind_t kind,
         const args_t &ref_args) {
-    cmp.set_threshold(0.f);
+    // The nvidia implementation has precision issues in some cases
```

Comment:
I wouldn't call it precision issues, wasn't our conclusion that this is differences in rounding modes that we can't change in cuDNN?

Reply:
I updated the comment in the latest commit.

Reply:
I don't see what the difference is? I'd say something like "different precision guarantees"

Reply:
Looks like I updated the check, not the comment. Updated the comment as suggested now, thanks.